config/default.yml

```diff
@@ -13,10 +13,13 @@ FactoryBot/AssociationStyle:
   Enabled: pending
   Safe: false
   Include:
-    - spec/factories.rb
-    - spec/factories/**/*.rb
-    - features/support/factories/**/*.rb
+    - "**/spec/factories.rb"
+    - "**/spec/factories/**/*.rb"
+    - "**/test/factories.rb"
```

Comment:
I wonder whether those should not be under the global 
```
FactoryBot:
  Include:
```
Most of the cops deal with the factory definitions. Then we have the ones that deal with factory usage in the tests - that could extend the include, but also we should look into their usage within minitest as well

Reply:
Might `FactoryBot/FactoryAssociationWithStrategy`, `FactoryBot/ConsistentParenthesesStyle`, `FactoryBot/RedundantFactoryOption`, and `FactoryBot/SyntaxMethods` become part of the following Include?
Therefore, even if you change the global configuration, I believe the above cops would still require the following settings. So, which ones should be considered as global settings?

```yaml
  Include:
    - "**/*_spec.rb"
    - "**/spec/**/*"
    - "**/test/**/*"
    - "**/features/support/factories/**/*.rb"
```

Reply:
Of the cops that are specifically for factory definitions, there are a handful that inherit the factory definition paths from the global includes (that already work with minitest factory definitions), and then there's this half that explicitly define them for the cop itself, while not including minitest factory definition paths.

It's honestly bad that these 4 cops even specify includes at all (and thus need to be overridden in downstream projects using minitest individually).

The global includes already list the factory definition paths for both RSpec and Minitest paths, and none of these cops should define includes at all, since they would already be properly inherited (unless explicitly overridden in downstream project configs per cop).